applications/tasks.py

```diff
@@ -8,53 +7,120 @@
+        organisation_identifier: str = getattr(organisation, "identifier", "") or ""
+        organisation_address: Address | None = getattr(organisation, "address", None)
+
+        reservation_details["description"] = (
+            translate_for_user(_("Core business"), application.user) + f": {organisation.core_business}"
+        )
+        reservation_details["reservee_organisation_name"] = getattr(organisation, "name", "")
+        reservation_details["reservee_id"] = organisation_identifier
+        reservation_details["reservee_is_unregistered_association"] = not organisation_identifier
+        reservation_details["reservee_address_street"] = getattr(organisation_address, "street_address", "")
+        reservation_details["reservee_address_city"] = getattr(organisation_address, "city", "")
+        reservation_details["reservee_address_zip"] = getattr(organisation_address, "post_code", "")
+
+    return reservation_details
 
 
 @app.task(name="generate_reservation_series_from_allocations")
 @SentryLogger.log_if_raises("Failed to generate reservation series from allocations")
-def generate_reservation_series_from_allocations(application_round_id: int) -> None:  # noqa: PLR0915
+def generate_reservation_series_from_allocations(application_round_id: int) -> None:
     allocations = AllocatedTimeSlot.objects.filter(
         reservation_unit_option__application_section__application__application_round=application_round_id,
     ).select_related(
         "reservation_unit_option__reservation_unit__origin_hauki_resource",
         "reservation_unit_option__application_section__application__application_round",
     )
 
-    # Find all closed opening hours for all reservation units where allocations were made.
-    # Check against these and not fully normalized opening hours since allocations can be made
-    # outside opening hours (as this system defines them), but should not be on explicitly
-    # closed hours, like holidays.
-    closed_time_spans: dict[int, list[TimeSpanElement]] = {}
-    for allocation in allocations:
-        resource = allocation.reservation_unit_option.reservation_unit.origin_hauki_resource
-        if resource is None or resource.id in closed_time_spans:
-            continue
-
-        application_round = allocation.reservation_unit_option.application_section.application.application_round
-        try:
-            client = ReservableTimeSpanClient(resource)
-        except ReservableTimeSpanClientError:
-            # Skip fetching opening hours if the resource indicates that it never has any opening hours.
-            # There is a slight chance that someone has updated the resource recently,
-            # and it didn't have any opening hour rules previously,
-            # but since there is a performance penalty, we still skip fetching for now.
-            continue
-
-        closed_time_spans[resource.id] = client.get_closed_time_spans(
```

Comment:
This method on the client is probably unused now, so it can be removed.

Reply:
Removed ✅